x-pack/packages/ml/trained_models_utils/src/constants/trained_models.ts

```diff
@@ -177,3 +177,37 @@ export type ElserVersion = 1 | 2;
 export interface GetModelDownloadConfigOptions {
   version?: ElserVersion;
 }
+
+export type InferenceServiceSettings =
```

Comment:
Just double checking here: For 8.13 onward, we also support cohere. Should that be added?

cohere: specify the text_embedding task type to use the Cohere service.
elser: specify the sparse_embedding task type to use the ELSER service.
hugging_face: specify the text_embedding task type to use the Hugging Face service.
openai: specify the text_embedding task type to use the OpenAI service.
elasticsearch: specify the text_embedding task type to use the E5 built-in model or text embedding models uploaded by Eland.

Reply:
Can we add cohere in a separate PR?